packages/shared/src/lib/tracking.ts

```diff
@@ -71,6 +116,31 @@ export const addRegionIdAndTrackingParamsToSupportUrl = (
         : baseUrl;
 };
 
+export const addRegionIdAndTrackingWithAmountsParamsToSupportUrl = (
```

Comment:
why do we need a new function for this? 
`amountsAbTestName/Variant` are optional and the logic looks the same

Reply:
Removed and fixed. Please review.